src/Maker/MakeEntity.php

```diff
@@ -376,6 +382,66 @@ private function askForNextField(ConsoleStyle $io, array $fields, string $entity
             $data['nullable'] = true;
         }
 
+        $data['apiFilter'] = null;
+
+        if (true === $apiOptions) {
+            $availableFilters = [
```

Comment:
Maybe could you store this list in a `public static` property, so users can add their own filters if needed. Also, I would use FQCN here (for the same reason).

Reply:
Done. Then see: https://github.com/symfony/maker-bundle/pull/619#discussion_r428528750